### ISSUE TITLE
Error in mvn dependency install 

### ISSUE THREAD

**Issue by ridmal** [ballerina-platform/ballerina-message-broker#214]
When I tried to execute mvn clean install command in message- broker root folder,  build failed in a broker-common module with the following error. ( In master branch) 
Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:2.17:check (validate) on project broker-common: Failed during checkstyle execution: There are 12 errors reported by Checkstyle 6.11.2 with https://raw.githubusercontent.com/wso2/code-quality-tools/master/checkstyle/checkstyle.xml ruleset. -> [Help 1]

**Comment by Asitha** [comment]
Hi Ridmal,

Can you share the 12 erros mentioned by the checkstyle plugin? This can give us an idea to find the root cause for this.
```
Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:2.17:check (validate) on project broker-common: Failed during checkstyle execution: There are 12 errors reported by Checkstyle
```

**Comment by ridmal** [comment]
Error log mentioned in checkstyle plugin.
F:\GSOC\WSo2\message-broker\modules\broker-common\src\main\java\org\wso2\broker\common\CommonConstants.java:22: error: First sentence should end with a period.
F:\GSOC\WSo2\message-broker\modules\broker-common\src\main\java\org\wso2\broker\common\data\types\Boolean.java:26: error: First sentence should end with a period.
F:\GSOC\WSo2\message-broker\modules\broker-common\src\main\java\org\wso2\broker\common\data\types\FieldValue.java:52: error: First sentence should end with a period.
F:\GSOC\WSo2\message-broker\modules\broker-common\src\main\java\org\wso2\broker\common\data\types\FieldValue.java:114: error: First sentence should end with a period.
F:\GSOC\WSo2\message-broker\modules\broker-common\src\main\java\org\wso2\broker\common\data\types\LongLongInt.java:24: error: First sentence should end with a period.
F:\GSOC\WSo2\message-broker\modules\broker-common\src\main\java\org\wso2\broker\common\data\types\ShortShortInt.java:24: error: First sentence should end with a period.
F:\GSOC\WSo2\message-broker\modules\broker-common\src\test\java\org\wso2\broker\common\data\types\FieldTableTest.java:32: error: Missing a Javadoc comment.
F:\GSOC\WSo2\message-broker\modules\broker-common\src\test\java\org\wso2\broker\common\data\types\FieldValueTest.java:27: error: Missing a Javadoc comment.
F:\GSOC\WSo2\message-broker\modules\broker-common\src\test\java\org\wso2\broker\common\data\types\LongIntTest.java:27: error: Missing a Javadoc comment.
F:\GSOC\WSo2\message-broker\modules\broker-common\src\test\java\org\wso2\broker\common\data\types\LongStringTest.java:27: error: Missing a Javadoc comment.
F:\GSOC\WSo2\message-broker\modules\broker-common\src\test\java\org\wso2\broker\common\data\types\ShortStringTest.java:27: error: Missing a Javadoc comment.
F:\GSOC\WSo2\message-broker\modules\broker-common\src\test\java\org\wso2\broker\common\FastTopicMatcherTest.java:30: error: First sentence should end with a period.

As a quick fix, I put   
 <properties>
        <checkstyle.skip>true</checkstyle.skip>
    </properties> in POM file and re buid the project. But other modules also have some checkstyle errors.

**Comment by Asitha** [comment]
Strange... Build is not failing for me even with those doc comment issues. 
Skipping checkstyle is not a good way to overcome this. Proper way would be to fix those doc issues.

Would be great if you can send a pull request with those doc issues fixed.

**Comment by ridmal** [comment]
ok, I am working on it.

**Comment by Asitha** [comment]
👍

**Comment by a5anka** [comment]
Can you share the mvn version you use?

**Comment by ridmal** [comment]
Apache Maven 3.3.9

**Comment by MaryamZi** [comment]
@ridmal  - The checkstyle issues were fixed with #334, #434 and #435. 

There seem to be some test failures on Windows, and we've created #436 to track and fix them.

Please do re-open this issue if you are still facing these issues.
